packages/hash/api/src/model/ontology/util.ts

```diff
@@ -13,14 +13,14 @@ export const getNamespaceOfAccountOwner = async (
   params: { ownerId: string },
 ) => {
   const namespace =
-    params.ownerId === systemAccountId
+    params.ownerId === systemOrgAccountId
       ? SYSTEM_ACCOUNT_SHORTNAME
```

Comment:
We don't need to be doing this now right? I mean it will be _slightly_ faster but I think it'd be more consistent to use the actual Org entity here.

Reply:
I agree in theory we don't need to do this, but when I tried removing it I discovered that `getNamespaceOfAccountOwner` is used before the system types are created so an error is thrown without this custom check for `systemOrgAccountId`.

By further restructuring things this can probably be avoided, but I'm not sure it's worth the hassle as this should be pretty robust. If you disagree I can spend more time looking into it.

Reply:
Ah I see, this seems to be another issue to do with the model classes. There are two options here I guess:
* Leave it as is and track it in a task as an issue to tackle next week alongside the model classes
* ~~Just remove the dependency on the model classes here similar to what you did for [this](https://github.com/hashintel/hash/pull/1596#discussion_r1040839366)~~

Actually it turns out the model classes are used in the type initializers, definitely not worth trying to restructure as a drive-by now.

Can you put a todo in here and associate it with the model classes, saying we need to do this because the initialization logic uses the model classes, which in turn uses this.